doc/src/tutorial/matrices.rst

```diff
@@ -151,7 +151,7 @@ raise it to the ``-1`` power.
     >>> N**-1
     Traceback (most recent call last):
     ...
-    ValueError: Matrix det == 0; not invertible.
+    sympy.matrices.common.NonInvertibleMatrixError: Matrix det == 0; not invertible.
```

Comment:
I don't think this needs the sympy.matrices.common part. The exception name and the message should be fine.

Reply:
Ok, any other changes?

Reply:
And the message as a string.